packages/sdks/web-component/src/lib/types.ts

```diff
@@ -64,6 +64,7 @@ export interface ScreenState {
   linkId?: unknown;
   sentTo?: unknown;
   clientScripts?: ClientScript[];
+  componentsState?: Record<string, string>;
```

Comment:
maybe add a comment - `// map component id to its optional state (e.g. 'hide')`

Reply:
Done